cpp/oneapi/dal/algo/triangle_counting/backend/cpu/relabel_kernels.hpp

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <memory>
+
+#include "oneapi/dal/algo/triangle_counting/common.hpp"
+#include "oneapi/dal/algo/triangle_counting/vertex_ranking_types.hpp"
+#include "oneapi/dal/backend/dispatcher.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/detail/threading.hpp"
+#include "oneapi/dal/common.hpp"
+#include "oneapi/dal/detail/policy.hpp"
+
+namespace oneapi::dal::preview {
+namespace triangle_counting {
+namespace detail {
```

Comment:
```suggestion
namespace backend {
```

Please apply to other cases

Reply:
Fixed